### ISSUE TITLE
How to find the absolution position of a collage on the page?

### ISSUE THREAD

**Issue by kentnek** [timjs/elm-collage#30]
I'm trying to implement an interactive collage that has a rectangular bounding rectangle, and a few elements on top of it. 

Using the given mouse events, I could find the absolute `Point` (relative to the client/page) of the events. Is it possible to get the mouse position relative to the origin of the bounding box instead? In other words, how to get the absolution position of the bounding box so that I can calculate the offset?

Thanks for the awesome library!

**Comment by 2426021684** [comment]
I would find this useful too

**Comment by timjs** [comment]
Hi! Sorry for being absent, didn’t work on this library for a long time because of lack of time and need. 

All shapes are translated to SVG. So if Elm has a way to get the absolute position of an SVG/HTML/DOM node, then it would be possible. However, I don’t think Elm allows you to do such a thing. If you know of a way to do this, PRs are always welcome!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by 2426021684** [comment]
https://package.elm-lang.org/packages/elm/browser/latest/Browser-Dom seems promising. I’ll let you know what I find.

**Comment by timjs** [comment]
Yes, at first sight that is a viable option, but they are async calls returning Tasks. I.e. users of the library should add certain messages to their app and route these back to their drawing logic. From a library design perspective, I find this a suboptimal solution, just like using [ports](https://github.com/timjs/elm-collage/issues/36#issuecomment-451648380). 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
